github/testground.yml

```diff
@@ -72,6 +74,8 @@ repositories:
       maintain:
         - bloxico
         - maintainers
+      push:
+        - celestia
```

Comment:
With GitHub Management in place I think `push` should be our default. @laurentsenta, can you think of any actions that might require higher permissions than that?

Reply:
I think `push` make sense,
we might want to give auth on `testground/testground` repo also; that's where issues are [centralized](https://github.com/testground/infra/#contribute).

Reply:
(thanks for the update)